votor/src/certificate_pool/vote_pool.rs

```diff
@@ -105,9 +102,8 @@ impl SimpleVotePool {
         true
     }
 
-    pub fn add_to_certificate(&self, output: &mut VoteCertificate) -> Result<(), CertificateError> {
-        output.aggregate(self.vote_entry.transactions.iter())?;
-        Ok(())
+    pub fn add_to_certificate(&self, output: &mut VoteCertificateBuilder) {
+        output.aggregate_unchecked(&self.vote_entry.transactions)
```

Comment:
Instead of calling a function which panics inside, can we use `output.aggregate` which returns an error and panic here? A bit easier for constructing tests.